source/developer/message-attachments.rst

```diff
@@ -124,12 +124,20 @@ And here is how it renders in Mattermost:
 
 .. image:: ../images/attachments-example.png
 
+Footer
+~~~~~~~
+
+``footer``: An optional line of text that will be displayed at the bottom of the attachment.
```

Comment:
Please add a note that this will be truncated if it's longer then 300 characters.

Reply:
@hanzei Done 👍